sympy/geometry/ellipse.py

```diff
@@ -418,20 +475,47 @@ def equation(self, x='x', y='y'):
 
         arbitrary_point : Returns parameterized point on ellipse
 
+        References
+        ==========
+
+        .. [1] https://math.stackexchange.com/questions/108270/what-is-the-equation-of-an-ellipse-that-is-not-aligned-with-the-axis
+        .. [2] https://en.wikipedia.org/wiki/Ellipse#Equation_of_a_shifted_ellipse
+
         Examples
         ========
 
         >>> from sympy import Point, Ellipse
         >>> e1 = Ellipse(Point(1, 0), 3, 2)
         >>> e1.equation()
         y**2/4 + (x/3 - 1/3)**2 - 1
+        >>> e1.equation(slope = 1)
+        (-x + y + 1)**2/18 + (x + y - 1)**2/8 - 1
+        >>> e2 = Ellipse(Point(0, 0), 4, 1)
+        >>> e2.equation(slope = 1)
+        (-x + y)**2/32 + (x + y)**2/2 - 1
 
         """
-        x = _symbol(x, real=True)
```

Comment:
x and y should stay here and not be below in the else

Reply:
@smichr Done. Please take a look.